### DISCUSSION TITLE
Sonoff Zigbee 3.0 Dongle Plus no longer /dev/ttyACM0  after recent OpenMediaVault update

### DISCUSSION THREAD

**Discussion by LuxMcDiver** [Koenkk/zigbee2mqtt#17596] [Devices]
Hi,
I am using thus dongle since a while on my Raspi4 with OMV, Docker and Portainer.
After an update of 46 OMV items the dongle cannot be found at the original location /dev/ttyACM0.
I now followed the procedure "Docker Stack device mapping" on https://www.zigbee2mqtt.io/guide/installation/02_docker.html#notes-for-raspberry-pi-users , but the device /dev/zigbee-serial  is not generated.

How can I troubleshoot this further?

Kind regards

Nico

**Comment by alexantao** [comment]
Hi, have you solved this ? 
I'm also triyng to make it work, but no juice.
Thanks.
